packages/muelu/src/Graph/MatrixTransformation/MueLu_AmalgamationFactory_def.hpp

```diff
@@ -101,13 +112,19 @@ namespace MueLu {
       } else {
         stridedblocksize = fullblocksize;
       }
+      // Correct for the storageblocksize
+      TEUCHOS_TEST_FOR_EXCEPTION(fullblocksize % storageblocksize != 0,Exceptions::RuntimeError,"AmalgamationFactory: fullblocksize needs to be a multiple of A->GetStorageBlockSize()");
```

Comment:
Oh, I understand.  `fullblocksize = getFixedBlockSize()`, which returns `numPDEs`.  It might make the code clearer to use a temporary variable `numPDEs`, e.g.,
```
   int numPDEs = stridedRowMap->getFixedBlockSize();
   fullblocksize = numPDES / storageblocksize;
```

Reply:
Yeah, but not easily (because of the stridedblocksize stuff.

I added a comment instead.